src/utils.ts

```diff
@@ -158,4 +159,67 @@ export function createDidPayload(verificationMethods: VerificationMethod[], veri
             authentication: verificationKeys.map(key => key.keyId)
         }
     )
+}
+
+export function createDidPayloadWithSignInputs(seed?: string, keys?: IKeyPair[]) {
+    if(seed && keys) throw new Error ('Only one of seed or keys should be passed as an argument')
+    
+    if(!keys) {
+        keys = [seed ? createKeyPairBase64(seed) : createKeyPairBase64()]
```

Comment:
Let's re-use `createKeyPairRaw` under the hood inside `createKeyPairBase64` and take `seed` as an optional argument on every encoding wrapper.

e.g. Generation in hex would be:
```typescript
  export function createKeyPairHex(seed?: string): IKeyPair {
    const keyPair = generateKeyPairRaw(seed)
    return {
        publicKey: toString(keyPair.publicKey, 'hex'),
        privateKey: toString(keyPair.secretKey, 'hex'),
    }
}
```

While `seed` condition is handled only at the initial `createKeyPairRaw()` execution as suggested above.

e.g. 
```typescript
  export function createKeyPairRaw(seed?: string): KeyPair {
    return seed ? generateKeyPairFromSeed(fromString(seed)) : generateKeyPair()
  }
```